lib/Mitigation.pm

```diff
@@ -0,0 +1,335 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2019 SUSE LLC
```

Comment:
The copyright year should be limited to 2019.

Reply:
Thanks,  fixed